content/applications/productivity/iot/config/https_certificate_iot.rst

```diff
@@ -0,0 +1,217 @@
+=======================
+HTTPS certificate (IoT)
+=======================
+
+What is HTTPS ?
+---------------
+
+*Hypertext Transfer Protocol Secure* (HTTPS) is the secure version of HTTP,
+which is the primary protocol used to send data between a web browser and
+a website. HTTPS is encrypted in order to increase security of data transfer.
+
+HTTPS uses an encryption protocol to encrypt communications. The protocol is
+called Transport Layer Security (TLS), although formerly it was known as
+Secure Sockets Layer (SSL).
+
+HTTPS occurs based upon the transmission of TLS/SSL certificates,
+which verify that a particular provider is who they say they are.
+
+.. note::
+   In this documentation and Odoo we will use the term "HTTPS certificate"
+   to define the fact that the SSL certificate is valid and allows HTTPS
+   connection
+
+
+Why is it needed ?
+------------------
+
+In order to communicate with certain network devices (in particular payment
+terminals), the usage of HTTPS is mandatory.
+In other words, if you don't have the HTTPS certificate, some
+devices won't be able to interact with the IoT-box.
+
+
+How to have it ?
+----------------
+
+The generation of this certificate is automatic.
+
+In practice, the IoT-box will send a specific request to `<https://www.odoo.com>`__
+that will send back the certificate if you are eligible
+
+IoT-box eligibility
+~~~~~~~~~~~~~~~~~~~
+
+ #. The database should be a **production** instance.
+    Not a copy, duplicate, staging or development.
+ #. Your Odoo subscription have an "IoT Box Subscription" line.
+    If you are not sure on the matter, contact your sales person or partner
+
+Troubleshoot
+------------
+
+If anything in the process of the HTTPS certificate generation or reception
+went wrong, a specific error will code be given on your IoT box homepage.
+
+
+ .. note::
+    In this state refreshing the IoT-box homepage will force to regenerate
```

Comment:
the formulation can be misleading IMHO (as the IoT box is not producing an error for real + we don't mention the HTTPS certificate). I will rephrase it otherwise

Reply:
Sounds good, I'll let you rephrase this. Let me know if you'd like me to re-review.